src/js/bootstrap-slider.js

```diff
@@ -1314,6 +1314,24 @@ const windowIsDefined = (typeof window === "object");
 								}
 								this.tickLabelContainer.style[styleMargin] = this.sliderElem.offsetWidth / 2 * -1 + 'px';
 							}
+
+							/* Set class labels to indicate tick labels are in the selection or selected */
+							this._removeClass(this.tickLabels[i], 'label-in-selection label-is-selection');
+							if (!this.options.range) {
+								if (this.options.selection === 'after' && percentage >= positionPercentages[0]) {
+									this._addClass(this.tickLabels[i], 'label-in-selection');
+								} else if (this.options.selection === 'before' && percentage <= positionPercentages[0]) {
```

Comment:
will give it a go. using this in production now so had to create a form and our own bower package. obvi not ideal so will get this done!